CMakeLists.txt

```diff
@@ -75,6 +75,10 @@ TRIBITS_ADD_OPTION_AND_DEFINE(
     LIST(APPEND DEVICE_LIST "<Cuda,CudaUVMSpace>")
   ENDIF()
 
+  IF(NOT CMAKE_CXX_USE_RESPONSE_FILE_FOR_OBJECTS)
```

Comment:
```
IF(NOT (DEFINED(CMAKE_CXX_USE_RESPONSE_FILE_FOR_OBJECTS)) OR (NOT CMAKE_CXX_USE_RESPONSE_FILE_FOR_OBJECTS))
```
etc.

Reply:
Not sure if `CMAKE_CXX_USE_RESPONSE_FILE_FOR_OBJECTS` is defined by default.  (CMake options have three possible states: `NOT DEFINED`, `FALSE`, and `TRUE`.)  The above will check for that.

Reply:
@mhoemmen I didn't realize that, thanks for the catch!